### DISCUSSION TITLE
close to open a file

### DISCUSSION THREAD

**Discussion by null-sudo** [nvim-neo-tree/neo-tree.nvim#499] [Q&A]
Hi!
how to close after opening a file (sorry for my English)

**Comment by cseickel** [comment]
There are several options. 

My favorite for this use case is to use `:Neotree current` to open neo-tree within the window you want to open the file in. Then it will naturally be replaced by the file you are opening.

The next best solution is to use `:Neotree float` to open a floating window which will automatically close when you open a file.

If you are really stuck on using a sidebar style that closes automatically, there is also this solution: https://github.com/nvim-neo-tree/neo-tree.nvim/wiki/Recipes#auto-close-on-open-file

**Reply by null-sudo** [reply]
Thank you!! It works perfect
